CHANGELOG.md

```diff
@@ -130,6 +130,10 @@ Adding a new version? You'll need three changes:
   expected to functionally affect routing, but may affect performance for
   some configurations.
   [#4934](https://github.com/Kong/kubernetes-ingress-controller/pull/4934)
+- KongIngress is deprecated and will be removed in a future release. Settings
```

Comment:
The changelog entry for this PR is not added?

Reply:
I think that @rainest's intention was to merge this PR to https://github.com/Kong/kubernetes-ingress-controller/pull/4989 branch. IMO #4989 is already big enough to not stuff more code in there. 

Let's wait for #4989 to be merged to main and we'll rebase this PR.